lib/main_ltp.pm

```diff
@@ -196,6 +196,10 @@ sub load_kernel_tests {
         loadtest 'boot_ltp';
         loadtest 'qa_test_klp';
     }
+    elsif (get_var('INSTALL_KLP_PRODUCT')) {
+        loadtest 'boot_ltp';
+        loadtest 'install_klp_product';
```

Comment:
Hm, after merging https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10627, `loadtest_kernel` needs to be used.